### DISCUSSION TITLE
how to change the link in the "Donate" option?

### DISCUSSION THREAD

**Discussion by libracrumbs** [KadeArchive/Kade-Engine#1721] [Q&A] [answered]
I saw it on vs Nonsense and i wanna do it too

**Comment by RokyDev** [comment, marked as answer]
go to MainMenuState.hx and at line 194, change "https://ninja-muffin24.itch.io/funkin" to whatever link you want

**Reply by libracrumbs** [reply]
Thank you!

**Reply by RokyDev** [reply]
np
